administration/intro.adoc

```diff
@@ -178,26 +178,21 @@ Using command line interface
 
 The following
 https://docs.openshift.com/container-platform/3.7/admin_guide/manage_scc.html[SCC]
-needs to be added prior KubeVirt deployment:
-
-- When you use the stable deployment flow:
+needs to be added prior KubeVirt deployment when using the stable deployment flow:
 ____________________________________
-Important: Starting with version v0.15.0 please replace `kubevirt-privileged` with `kubevirt-handler`
+Important: Before v0.15.0 release, please replace `kubevirt-handler` with
+`kubevirt-privileged`.
 ____________________________________
 [source,bash]
 ----
-$ oc adm policy add-scc-to-user privileged -n kubevirt -z kubevirt-privileged
+$ oc adm policy add-scc-to-user privileged -n kubevirt -z kubevirt-handler
 $ oc adm policy add-scc-to-user privileged -n kubevirt -z kubevirt-controller
 $ oc adm policy add-scc-to-user privileged -n kubevirt -z kubevirt-apiserver
 ----
 
-- When you use the operator flow:
-[source,bash]
-----
-$ oc adm policy add-scc-to-user privileged -n kubevirt -z kubevirt-operator
```

Comment:
To my best knowledge this is still needed. The operator can't add the kubevirt-* accounts to the privileged SCC without being in that SCC itself.

Reply:
I've deployed the operator on OCP4 several days ago and it didn't need the SCC part, but I will recheck.